charts/lagoon-logs-concentrator/templates/_helpers.tpl

```diff
@@ -61,3 +61,14 @@ Create the name of the service account to use
 {{- default "default" .Values.serviceAccount.name }}
 {{- end }}
 {{- end }}
+
+{{/*
+Get HorizontalPodAutoscaler API Version
+*/}}
+{{- define "lagoon-logs-concentrator.hpa.apiVersion" -}}
+  {{- if and (.Capabilities.APIVersions.Has "autoscaling/v2") (semverCompare ">= 1.23-0" .Capabilities.KubeVersion.Version) -}}
```

Comment:
Why do we need two conditional checks here? If `autoscaling/v2` is available, I would have thought the version of k8s is irrelevant?

Reply:
in that case maybe just a comment will do?

Reply:
fixed